Configuration/Eras/python/Era_Run2_2016_UPC_cff.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run2_2016_cff import Run2_2016
+from Configuration.ProcessModifiers.egamma_lowPt_exclusive_cff import egamma_lowPt_exclusive
+from Configuration.Eras.Modifier_highBetaStar_2018_cff import highBetaStar_2018
+from Configuration.Eras.Modifier_run3_upc_cff import run3_upc
+
+Run2_2016_UPC = cms.ModifierChain(Run2_2016, egamma_lowPt_exclusive, highBetaStar_2018, run3_upc)
```

Comment:
Unrequested comment, from a non-reviewer. Perhaps these modifier are generic enough to be renamed without directly referencing any data taking year? It is somewhat confusing to define a 2016 era with 2018 and run3 ingredients in it.

Reply:
I could rename "highBetaStar_2018" into "highBetaStar", and "run3_upc" into "upc", if that helps

Reply:
Changed the modifier "highBetaStar_2018" to "highBetaSta" in 5573b8fef99e367c06a00d9fee1ad4f47804076b